src/components/controls/ProbeKeySelector.svelte

```diff
@@ -4,6 +4,7 @@
   import { tooltip as tooltipAction } from '@graph-paper/core/actions';
 
   export let options;
+  options.sort();
```

Comment:
@robhudson since we want to sort options in 2 locations, do you think it's better that we just sort it here instead?

Reply:
If we need it sorted in both places? I wasn't sure what that 2nd loop actually did at first glance. But sorting it here in the JS portion of the svelte code works too. Whatever you think is best!